### PR TITLE
refactor: standardization add children

### LINKED ISSUE #104: Beagle standardization - Children
**Describe the problem**
In the default components of the Beagle, there are several nomeclatures for your child views.

**Expected behavior**
Change in Beagle whatever is rows, pages, etc. for `children` for the purpose of semantic coherence or whatever is more than one child should be called children.

### PR DESCRIPTION
**Pull request by luisoliveirazup**
closes #104 

BREAKING CHANGE: ListView: rename rows to children, PageView: rename pages to children, TabView: rename tabItems to children, TabItem: rename content to child

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Tests have failed, see below for more information.</td>
    </tr>
  </tbody>
</table>

### Tests: 

Name | Classname | Time|
--- | --- | ---|
make_should_return_moshi_to_deserialize_a_PageView | br.com.zup.beagle.android.data.serializer.BeagleMoshiTest | 0.006|
make_should_return_moshi_to_deserialize_a_ListView | br.com.zup.beagle.android.data.serializer.BeagleMoshiTest | 0.001|


<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">The title of PR not follow the rules of: https://www.conventionalcommits.org/en/v1.0.0/</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by pedropnaves** [comment]
Please do not merge until the alignment between all platforms. I'm afraid about ListView

**Comment by luisoliveirazup** [comment]
> E o `navigationBarItems`atributo dentro da tela `NavigationBar`também não deveria ser renomeado `children`?

File: iOS/Sources/BeagleUI/Sources/Components/TabBar+Renderable/TabView.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ import BeagleSchema
 extension TabView: ServerDrivenComponent {
 
     public func toView(renderer: BeagleRenderer) -> UIView {
-        let model = TabViewUIComponent.Model(tabIndex: 0, tabViewItems: tabItems)
+        let model = TabViewUIComponent.Model(tabIndex: 0, tabViewItems: children)
         let tabView = TabViewUIComponent(model: model)
 
         // TODO: use style in BeagleRenderer
```
Comment (botbeaglezup) [inline iOS/Sources/BeagleUI/Sources/Components/TabBar+Renderable/TabView.swift:26]: <table data-meta="generated_by_ios">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/66e509d76bd5a7171c35ef64ee28cd716cb0ef43/iOS/Sources/BeagleUI/Sources/Components/TabBar+Renderable/TabView.swift#L26"></span>TODOs should be resolved (use style in BeagleRenderer).<br />
<code>todo</code> <code>TabView.swift:26</code></td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by tuliopereirazup** [comment]
@pedropnaves do can we continue with this merge?

**Comment by pedropnaves** [comment]
> @pedropnaves can we continue with this merge?

LGTM

**Pull request merged**